src/parkapi_sources/converters/kienzler/converter.py

```diff
@@ -90,7 +90,7 @@ def _get_kienzler_parking_sites(self) -> tuple[list[KienzlerInput], list[ImportP
                 )
         return kienzler_item_inputs, errors
 
-    def _request(self) -> list[dict]:
+    def _request_kienzler(self) -> list[dict]:
```

Comment:
Just out of curiosity: was the renaming done because `_request_kienzler` is a better name, or is there another reason, like `_request` already existing for something else?

Reply:
Because we have two requests there, one to Kinezler and one to the GeoJSON. And confused myself multiple times because I had to look in the method which data is requested.